src/app/send-tx/send-tx.component.ts

```diff
@@ -43,6 +43,12 @@ export class SendTxComponent implements OnInit {
 
   updateBalance(): void{
     let val = this.txForm.get('privateKey').value;
+
+    if (val.indexOf('0x') !== 0) {
```

Comment:
This is probably right, but is there ever a chance of 0x being somewhere in the middle?

Reply:
it wouldn't matter. if 0x is not the first two characters it would simply prepend 0x.